gbfs.md

```diff
@@ -190,22 +229,90 @@ stations          | Yes       | Array that contains one object per station in th
 \- region_id       | Optional  | ID of the region where station is located (see [system_regions.json](#system_regionsjson))
 \- post_code       | Optional  | Postal code where station is located
 \- rental_methods  | Optional  | Array of enumerables containing the payment methods accepted at this station. <br />Current valid values (in CAPS) are:<br /><ul><li>KEY _(i.e. operator issued bike key / fob / card)_</li> <li>CREDITCARD</li> <li>PAYPASS</li> <li>APPLEPAY</li> <li>ANDROIDPAY</li> <li>TRANSITCARD</li> <li>ACCOUNTNUMBER</li> <li>PHONE</li> </ul> This list is intended to be as comprehensive at the time of publication as possible but is subject to change, as defined in [File Requirements](#file-requirements) above
-\- capacity        | Optional  | Number of total docking points installed at this station, both available and unavailable
+\- capacity        | Optional  | Number of total docking points installed at this station, both available and unavailable, regardless of what vehicle types are allowed at each dock. This field should only be used if every dock at the station is able to accept every vehicle type used in the system.
+\- vehicle_type_capacity        | Optional  | An object where each key is a vehicle_type_id as described in [vehicle_types.json](#vehicle_typesjson) and the value is a number representing the total docking points installed at this station, both available and unavailable for the specified vehicle type.
```

Comment:
Do we need to enable a hierarchy or groups of vehicle types? e.g. are there cases where there are various defined electric bikes with different amenities, and we just need to specify capacity for electric bikes in general?

Reply:
I don't think this is necessary as it is possible to determine the form factor and additional information about a vehicle type by looking up the data associated with a particular vehicle type ID.

Reply:
Recently and number of cities ( DC, Bay Area, Portland etc.) have taken steps to introduce various types of adaptive cycles to their bike share fleets.  Additionally there are a number of examples of shared cargo bikes now in Europe. How would these types of vehicles be differentiated within the form factor and vehicle type? Existing examples are:

- Upright handcycle
- Recumbent handcycle
- Recumbent leg trike
- Recumbent trike tandem
- Side-by-side tandem bicycle
- Cargo bicycle
- Cargo tricycle

Reply:
I think the intent of the _form_factor_ field is to have a very general classification and avoid making up a potentially endless hyper-specific list of enumerations of what a vehicle's "type" is. Therefore, all of those vehicles @mplsmitch listed would each have a _form_factor_ value of `bicycle`. For now, the place to further explain details about the vehicle would be in the _description_ field of a vehicle type object in the vehicle_types.json file. In my initial PR, I proposed to add many more fields with additional places to differentiate vehicles (such as adding pictures which @sven4all [brought up again](https://github.com/NABSA/gbfs/pull/136#discussion_r328438810)), but I removed them in order to try to make a minimum viable pull request to merge in.